js/ugrd-dropdown.js

```diff
@@ -0,0 +1,59 @@
+( function( $) {
```

Comment:
haha, that punk. I still have JS linting off because of the other script. :) Thanks!